components/BlurhashCanvas.vue

```diff
@@ -25,21 +25,14 @@ export default Vue.extend({
       default: 1
     }
   },
-  watch: {
```

Comment:
Shouldnt we still have a `watch` method in case the hash changes

Reply:
@Camc314 That's why I asked, but when?

Reply:
@Camc314 That's why I asked, but when?

Reply:
🤷‍♀️ I guess in the case that a library filter is applied/removed, and a new card is not generated, but a new has is recieved

Reply:
It makes the canvas reactive to changes above it :)

DOM recycling, for example, just switches the properties of cards instead of destroying/recreating them. So anything that would lead to a change needs to react.

Here, if the item of the card changes, the hash changes, so the canvas would have to re-draw with the new pixel array.

Reply:
Also, once we have the websockets going, we'll get item updates notifications, so we can re-fetch a single item and update its card in place :yum:

Reply:
@MrTimscampi Keeping this as it was originally.